scalajs-tutorial-fastopt.html

```diff
@@ -1,11 +1,14 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>The Scala.js Tutorial</title>
-  </head>
-  <body>
-    <!-- Include Scala.js compiled code -->
-    <script type="text/javascript" src="./target/scala-2.13/scala-js-tutorial-fastopt/main.js"></script>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>The Scala.js Tutorial</title>
+</head>
+
+<body>
+  <!-- Include Scala.js compiled code -->
+  <script type="text/javascript" src="./target/scala-3.0.0/scala-js-tutorial-fastopt/main.js"></script>
```

Comment:
I was expecting something like "./target/scala-3.0" instead of "./target/scala-3.0.0"
@sjrd Was this change intentional?

Reply:
This entirely depends on the default value of sbt's `crossTarget` setting, so you'd have to ask that to sbt.